### PR TITLE
Revise the singular and plural forms of variables

### LINKED ISSUE #203: Revise the singular and plural forms of variables
The variables using for media is not correct and may mislead in development.

https://github.com/desmos-labs/desmos/blob/ed22aae536446e39f46aa109663ea02236f00d13/x/posts/internal/types/models/common/post_media.go#L14

`Media` is the plural form of the word `Medium`. Should we change the type to `PostMedium` and the variable of the slice `PostMedia` instead of `PostMedias`, and the object in the msg type `Media` instead of `Medias`?

### PR DESCRIPTION
**Pull request by bragaz**
## Description
This PR renamed the `PostMedia` to `Attachment` in order to fix the wrong singular and plural forms of variables.
Closes #203. 
## Checklist
- [x] Targeted PR against correct branch.
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote unit tests.
- [ ] Wrote integration tests (simulation & CLI). 
- [x] Updated the documentation. 
- [x] Added an entry to the `CHANGELOG.md` file.
- [x] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

File: x/posts/simulation/utils.go [hunk 1]
```diff
@@ -65,7 +65,7 @@ type PostData struct {
 	Subspace       string
 	CreationDate   time.Time
 	OptionalData   map[string]string
-	Medias         types.PostMedias
+	Medias         types.Attachments
```
Comment (RiccardoM) [inline x/posts/simulation/utils.go:68]: Probably we can rename this field as well from `Medias` to `Attachments`

File: x/posts/types/models/common/attachment.go [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+	"github.com/desmos-labs/desmos/x/commons"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// ---------------
+// --- Attachment
+// ---------------
+
+type Attachment struct {
```
Comment (RiccardoM) [inline x/posts/types/models/common/attachment.go:16]: Can we add a small documentation telling what an `Attachment` represents?

**Comment by RiccardoM** [comment]
@bragaz looks like the docs links are not setup properly :thinking:

**Comment by bragaz** [comment]
I'm fixing them rn

**Comment by RiccardoM** [comment]
@bragaz conflicts should be resolved, then we should be good to go

**Pull request merged**